docs/VnVPlan/VnVPlan.tex

```diff
@@ -276,64 +276,342 @@ \subsection{Software Validation Plan}
+relevant SRS  \citep{SRS}. 
 
 \subsection{Tests for Functional Requirements}
 
-\wss{Subsets of the tests may be in related, so this section is divided into
-  different areas.  If there are no identifiable subsets for the tests, this
-  level of document structure can be removed.}
+Each test section addresses a particular initial condition and input set. 
+Therefore, each test has a particular set of outputs that the system should 
+produce for it. A functional requirement of section 9 in the SRS \citep{SRS}
+is only associated with a test if the output set of the test corresponds to 
+the fit criterion of that functional requirement, and the fit criterion should 
+be an observable component of a system state, such as a display of a return 
+item or notification presented. Through this association, every functional 
+requirement is provided a test that is capable of verifying it, thereby ensuring 
+the test areas fully cover the functional requirements.
 
-\wss{Include a blurb here to explain why the subsections below
-  cover the requirements.  References to the SRS would be good here.}
+\subsubsection{Test Area 1}
 
-\subsubsection{Area of Testing1}
+Applies to all FRs involving input of the system necessary for the plagiarism 
+analysis. This currently covers FR-1 in subsection 9.1 of the SRS \citep{SRS}.
 
-\wss{It would be nice to have a blurb here to explain why the subsections below
-  cover the requirements.  References to the SRS would be good here.  If a section
-  covers tests for input constraints, you should reference the data constraints
-  table in the SRS.}
-		
-\paragraph{Title for Test}
+\paragraph{Input For Analysis Test}
 
 \begin{enumerate}
 
-\item{test-id1\\}
+\item{test-FR1-1\\}
 
-Control: Manual versus Automatic
+Control: Manual.
 					
-Initial State: 
+Initial State: system is active, spot for giving system code snippet is open
+with zero or more existing snippets currently added.
 					
-Input: 
+Input: code snippet file(s).
 					
-Output: \wss{The expected result for the given inputs.  Output is not how you
-are going to return the results of the test.  The output is the expected
-result.}
+Output: System signifies input was received and continues activity without 
+error, awaiting further action directives (such as giving more files or 
+initiating analysis).
 
-Test Case Derivation: \wss{Justify the expected value given in the Output field}
+Test Case Derivation: The system will look to continually receive code 
+snippets from user as part of a necessary step to initiate plagiarism analysis.
 					
-How test will be performed: 
+How test will be performed: system will be opened in the appropriate spot for 
+receiving code and receive files in different amounts, one at a time, 
+and will be inspected for failure in this process (types: dynamic, functional).
 					
-\item{test-id2\\}
 
-Control: Manual versus Automatic
+\end{enumerate}
+
+\subsubsection{Test Area 2}
```

Comment:
rather than saying test area 2, give it a nicer and more descriptive name

Reply:
I'll change the Test Area names to the Test titles, Lucas said it was alright to exchange one of them since it was redundant.